src/SmartComponents/RunTaskModal/ConversionTaskInputParameters.js

```diff
@@ -0,0 +1,197 @@
+          return { key: param.key, value: newValue };
+        } else {
+          return param;
+        }
+      });
+    });
+  };
+  const toBool = (value) => {
+    return value.toLowerCase() === 'true' || value === '1';
+  };
+
+  const readMoreLink = (
+    <a
+      href="https://www.redhat.com/en/blog/announcing-4-years-extended-life-cycle-support-els-red-hat-enterprise-linux-7"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Read more about ELS for RHEL 7.
+    </a>
+  );
+
+  const convert2RHELSupportLink = (
+    <a
+      href="https://access.redhat.com/support/policy/convert2rhel-support"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Convert2RHEL Support Policy
+    </a>
+  );
+
+  const inPlaceSupportLink = (
+    <a
+      href="https://access.redhat.com/support/policy/ipu-support"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      In-place upgrade Support Policy
+    </a>
+  );
+
+  const analysisTitleSection = (
+    <TextContent>
+      <Text component={TextVariants.h4}>
+        Analyze conversion to RHEL 7 without Extended Lifecycle Support (ELS)
+      </Text>
+      <Text component={TextVariants.p}>
+        By default, the task analyzes a conversion of your CentOS Linux 7 to a
+        supported RHEL 7 system with the latest security patches and updates via
+        ELS. {readMoreLink}
+      </Text>
+    </TextContent>
+  );
+
+  const convertTitleSection = (
+    <TextContent>
+      <Text component={TextVariants.h4}>
+        Convert to RHEL 7 without Extended Lifecycle Support (ELS)
+      </Text>
+      <Text component={TextVariants.p}>
+        By default, the task converts your CentOS Linux 7 to a supported RHEL 7
+        system with the latest security patches and updates via ELS.{' '}
+        {readMoreLink}
+      </Text>
+    </TextContent>
+  );
+
+  return (
+    <Form className="pf-v5-u-pb-lg">
+      {slug.includes('analysis') ? analysisTitleSection : convertTitleSection}
+
+      <Checkbox
+        id={elsDisabled.key}
+        label={elsDisabled.title}
+        isChecked={toBool(elsDisabledValue)}
+        onChange={(_event, value) => {
+          const newValue = value ? 'True' : 'False';
+          setElsDisabledValue(newValue);
+          updateParameters(elsDisabled, newValue);
+        }}
+      />
+      <TextContent>
+        <Text
+          component={TextVariants.p}
+          style={{
+            fontSize: '14px',
+            color: '#6a6e73',
+            paddingLeft: '20px',
+            marginTop: '-20px',
+          }}
+        >
+          If you plan to upgrade to RHEL 8 right after the conversion, you may
+          opt not to use the ELS subscription. Note that the conversion and the
+          subsequent upgrade without an ELS subscription come with a limited
+          support scope per the {convert2RHELSupportLink} and the{' '}
+          {inPlaceSupportLink}.
+        </Text>
```

Comment:
This must be provided simply to `Checkbox` thru the `description` prop: https://www.patternfly.org/components/forms/checkbox/#checkbox-with-description

Reply:
Ok, so I was able to create a description with links in it :tada: